meshery

```diff
@@ -127,42 +127,77 @@ unzip ${PWD}/meshery.zip -d ${PWD}/$temporary_dir_name
 install_dir=${MESHERY_INSTALL_DIR}
 if [ -z "$install_dir" ]
 then
-    install_dir="/usr/local/bin"
+install_dir="/usr/local/bin"
```

Comment:
@devansh42 these outdentations (if I can make up that word) seem undesirable given that these statements are executed _if_ the preceding line is true. Could you have a look at reverting this formatting?

Reply:
I'd surely add these changes to the next commit.

Reply:
Ok, signal when done.